northstar/src/runtime/state.rs

```diff
@@ -484,62 +484,67 @@ impl State {
         Ok(None)
     }
 
+    async fn uninstall_resource(
+        &mut self,
+        name: &str,
+        version: &Version,
+    ) -> result::Result<(), Error> {
+        // TODO check if resource still needed
+        self.resources.remove(&(name.to_owned(), version.clone()));
+        Err(Error::ApplicationNotFound)
+    }
+
+    async fn uninstall_app(&mut self, name: &str) -> result::Result<(), Error> {
+        println!("try to uninstall {}", name);
```

Comment:
Debug print that should be removed or used `debug!` according to the northstar log style.